### ISSUE TITLE
Prop types aren’t checked if JSX is returned in a sequential expression (e.g., `return 0, <div/>`)

### ISSUE THREAD

**Issue by mikol** [yannickcr/eslint-plugin-react#2800]
A sequential expression using the comma operator (`,`) evaluates each of its operands (from left to right) and returns the value of the last operand. If the value of the last operand is JSX, it should be validated. However, sequential expressions are not handled by `eslint-plugin-react`. For example, the `name` prop in this contrived example isn’t validated:

```jsx
import React from 'react'

export default function Component(props) {
  return 0,
  <div>
    Hello, { props.name }!
  </div>
}
```

**Comment by ljharb** [comment]
Fair, this is a bug, but what kind of crap code uses the comma operator??
